dockerfiles/Dockerfile.bor

```
@@ -3,7 +3,7 @@ FROM golang:1.16 as builder
 RUN apt-get update && apt-get install -y build-essential curl make gcc g++ git fio cmake clang-3.9 tzdata
 
 ARG BOR_VERSION="0.3.0"
```

nit: let's update the version tag as well
```suggestion
ARG BOR_VERSION="0.2.10"
```